README.md

```diff
@@ -98,6 +98,7 @@ Options:
   [--server], [--no-server]  # Start standalone web server on port 8080
   [--port=PORT]              # Set port for --server option
   [--host=HOST]              # Set host for --server option
+  [--watch], [--no-watch]    # Reload API documentation when API Description Document changed
```

Comment:
imho it is not needed - it is std for https://github.com/erikhuda/thor/wiki and the same as for other args